brjs-sdk/sdk/libs/javascript/br-test/src/br/test/ViewFixture.js

```diff
@@ -291,11 +291,13 @@ ViewFixture.prototype._getViewElements = function(propertyName) {
 };
 
 /** @private */
-ViewFixture.prototype._verifyOnlyOneElementSelected = function(elements, selector) {
+ViewFixture.prototype._verifyOnlyOneElementSelected = function(elements, selector, propertyName, value) {
 	if (elements.length === 0) {
-		throw 'No view element found for "' + selector + '"';
+		throw 'No view element found for "' + selector + '". Processing property "' + propertyName +
```

Comment:
here 'selector' isn't a selector, its a view handler. Can you change the arg name to viewHandler